aiida_quantumespresso/parsers/cp.py

```diff
@@ -45,10 +45,10 @@ def parse(self, **kwargs):
             return self.exit(self.exit_codes.ERROR_MISSING_XML_FILE)
 
         # Let's pass file handlers to this function
-        out_dict, _raw_successful = parse_cp_raw_output(
-            out_folder.open(stdout_filename), out_folder.open(xml_files[0]),
-            out_folder.open(self.node.process_class._FILE_XML_PRINT_COUNTER_BASENAME)
-        )
+        output_stdout = out_folder.get_object_content(stdout_filename)
+        output_xml = out_folder.get_object_content(xml_files[0])
```

Comment:
Is there a particular reason to use `get_object_content` instead of 

```python
with out_folder.open(stdout_filename) as stdout_handle:
    parse_cp_raw_output(stdout_handle, ...)
```
If the files are large, I'd think the latter could potentially be more efficient (depending on what `parse_cp_raw_output` does) because it doesn't need to load the whole file into memory at once.

Or are these files generally small enough that it doesn't matter?

Reply:
It wouldn't matter because `parse_cp_raw_output` would load them into memory anyway and has to because it cannot parse it in a streaming manner. At least not currently.

Reply:
Ok, yeah in that case it doesn't matter 👍